src/css.js

```diff
@@ -113,6 +113,18 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( !support.reliableFullscreenBox && window.top !== window.self &&
+		document.msFullscreenElement ) {
```

Comment:
@markelog In this PR I think adding the integration test would be enough. I don't like the idea of creating a support test function that works differently than all others; there is a lot more code that maybe could be refactored this way ([checkClone](https://github.com/jquery/jquery/blob/842958e7aecd0d75a7ee9e2aaec83457701aa2f3/src/manipulation.js#L126), [clearCloneStyle](https://github.com/jquery/jquery/blob/842958e7aecd0d75a7ee9e2aaec83457701aa2f3/src/css.js#L233), [pixelMarginRight](https://github.com/jquery/jquery/blob/842958e7aecd0d75a7ee9e2aaec83457701aa2f3/src/css/curCSS.js#L30) etc.) and it would be inconsistent to change just one (I'm not sure I like this direction either way but I don't think the discussion belongs to this PR/issue).

The only additional things I'm OK with doing in this PR are adding a HTML page with an integration test and optionally removing this support test altogether and just checking `if ( document.msFullscreenElement && window.top !== window )` as the test doesn't buy us a lot, indeed. Everything else I'd like to discuss separately as they seem to be refactorings that affect way more than just code in this PR.